handbook/engineering/README.md

```diff
@@ -644,6 +644,36 @@ The following rituals are engaged in by the directly responsible individual (DRI
 | Release ritual                | Every three weeks   | Go through the process of releasing the next iteration of Fleet.                                                                       | Zach Wasserman |
 | Create patch release branch   | Every patch release | Go through the process of creating a patch release branch, cherry picking commits, and pushing the branch to github.com/fleetdm/fleet. | Luke Heath     |
 
+## Fleet Sandbox On Call
+
+On call for Sandbox will happen in shifts of 1 week. The people involved in them will be:
+
+First responders:
```

Comment:
Please use "oncall". We are currently using both "on-call" and "oncall" (separately from this PR, we should standardize to "oncall"). I'd like to avoid adding a third variation.

Reply:
I went ahead and changed them all, it wasn't that much.